src/main/java/org/jenkinsci/test/acceptance/recorder/TestRecorderRule.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2014 Red Hat, Inc.
```

Comment:
The funny thing is that I copied this from one ATH class, so it is somewhere too.